tools/format_python_tools.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+import fnmatch
+import os
+import sys
+
+from yapf.yapflib.yapf_api import FormatFile
+
+
+def collect_files():
+  """Collect all Python files in the tools directory."""
+  matches = []
+  for root, dirnames, filenames in os.walk('.'):
```

Comment:
It would be nice if, like check_format.py, this could be applied to files specified on the command-line. That could also be eventually incorporated into the git push hooks, which is pretty painless if it's just on the files you are pushing.

Can you add a TODO for this?